rpc/wrtc_server.go

```diff
@@ -43,6 +44,51 @@ type webrtcServer struct {
 
 	onPeerAdded   func(pc *webrtc.PeerConnection)
 	onPeerRemoved func(pc *webrtc.PeerConnection)
+
+	counters struct {
+		PeersConnected       atomic.Int64
+		PeersDisconnected    atomic.Int64
+		PeerConnectionErrors atomic.Int64
+		HeadersProcessed     atomic.Int64
+		// CallTicketsAvailable is technically a guage. It increments and decrements. Rather than
+		// just going up.
```

Comment:
Are you calling this out because you would recommend monotonically-increasing metrics instead of ones that increment and decrement?